thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/performanceEvaluation/PrecisionByTimePerformanceEvaluation.java

```diff
@@ -0,0 +1,63 @@
+package com.linkedin.thirdeye.anomalydetection.performanceEvaluation;
```

Comment:
Please write few comments in your pull request to explain what you try to achieved here and how you do it. The commit 'title' sometimes is not sufficient, especially for this scale of code change (adding).